### ISSUE TITLE
Support for channel_topic event message

### ISSUE THREAD

**Issue by jahell** [slackapi/java-slack-sdk#666]
Hi there,

I'm trying to create an event listener for the `channel_topic` event (https://api.slack.com/events/message/channel_topic), however that event does not currently seem supported. 

Have not dug too deep into the code, but if it's a quick fix would be highly appreciated :)

### Category (place an `x` in each of the `[ ]`)

Not exactly sure what all the surface area of this is. 

* [ ] **bolt** (Bolt for Java)
* [ ] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [ ] **slack-api-client** (Slack API Clients)
* [ ] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

### Requirements

Please make sure if this topic is specific to this SDK. For general questions/issues about Slack API platform or its server-side, could you submit questions at https://my.slack.com/help/requests/new instead. :bow:

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to the those rules.

**Comment by seratch** [comment]
Hi @jahell, thanks for asking this! It has been supported since v1.4.1. Subscribe to the `MessageChannelTopicEvent` class:

```java
app.event(MessageChannelTopicEvent.class, (event, ctx) -> {
  ctx.logger.info("channel topic - {}", event);
  return ctx.ack();
});
```
See the PR https://github.com/slackapi/java-slack-sdk/pull/650 for details.

**Comment by jahell** [comment]
Ahh, sorry. Didn't realize that we were behind this much. Thanks for the quick response

**Comment by jahell** [comment]
@seratch Upgrading to `v.1.5.2` fixed this for us, but curious about something since when looking a bit closer it seems like our version was `v.1.5.0-M1` which sounds like it should have had the code given that it was introduced in `v.1.4.1`. I also can't seem to find `v.1.5.0-M1` in the tags anymore. Was there something wrong with that release that it got removed?

**Comment by seratch** [comment]
@jahell We should have cut off v1.5.1-M1 tag but I forgot to do so. The milestone version was shipped for early beta users of Socket Mode #633 . The release date of the version was Dec 2, which is prior to v1.4.1 release (on Dec 17).

* https://search.maven.org/artifact/com.slack.api/bolt-socket-mode/1.5.0-M1/jar
* https://search.maven.org/artifact/com.slack.api/bolt/1.4.1/jar

**Comment by jahell** [comment]
thanks for explaining.
